### DISCUSSION TITLE
Managing multiple Accounts on mobile

### DISCUSSION THREAD

**Discussion by paetzenstein** [community/community#39693] [Enterprise]
Hi,

is it possible to be logged in with multiple accounts on GitHub-Mobile?

For Example: I want to be logged in with my personal account and work account and switch seamless between them.

Right now, I only see the option to add a GitHub Enterprise-Server Account (selfhosted).

Did I miss something?

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.89.0 (83472182)
Device: iPhone 13 (iOS 16.1.1)
</details>

**Comment by ossanna16** [comment]
Hi there @paetzenstein and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by jsoref** [comment]
Please search before opening a new discussion. There are probably a dozen copies of this request.

https://github.com/orgs/community/discussions/39511
https://github.com/orgs/community/discussions/38185
https://github.com/orgs/community/discussions/3943
https://github.com/orgs/community/discussions/19989
https://github.com/orgs/community/discussions/3969

Some have comments from people reminding that on average if you're requesting this feature, you're probably using GitHub in violation of its terms of service (I found at least two cases where this was noted, one by me and one by someone else).

**Reply by paetzenstein** [reply]
Hi @jsoref ,

They added this feature for GitHub Mobile four months ago
So I guess there shouldn't be a ToS Violation, because that feature would be a Violation itself.

**Reply by jsoref** [reply]
Yes, I saw. It's also available on desktop.

I haven't spent time reading the terms of service to see if they've been updated.

Note that a paid account would be within the terms, just rare.

**Comment by eliperkins** [comment]
Hey @paetzenstein and folks, we've shipped functionality this week to allow you to log in with multiple accounts, like your personal and work accounts, at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/).

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.
